modules/base/manifests/init.pp

```diff
@@ -3,11 +3,21 @@
 # Pull in a standard set of modules for all machines. Used everywhere, by
 # both `govuk::node::s_base` and `govuk::node::s_development`.
 #
-class base {
+# [*esm_repo*]
+#   Wether or not to install ESM repo.
+#
+
+class base (
+  $esm_repo = false,
+) {
   class { '::apparmor':
     package_ensure => present,
   }
 
+  if $esm_repo {
```

Comment:
I think we need a filter for the last precise machines (licensify-lb) here while they last : /

Reply:
It should be fine, esm_repo is set to true in common.trusty.yaml, so only for the machines running trusty